metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-rest-env.xml

```diff
@@ -91,16 +91,10 @@
     <property>
         <name>metron_temp_grok_path</name>
         <description>Temporary local file path where grok patterns are written during testing</description>
```

Comment:
Description should be "Temporary HDFS file path" instead of "Temporary local file path"

Reply:
Done